docs/kb/semgrep-ci/scan-monorepo-in-parts.md

```diff
@@ -0,0 +1,54 @@
+---
+description: How to scan a monorepo in parts for better CI performance and clearer findings organization
+tags:
+  - monorepo
+  - optimization
+  - performance
+  - ci
+---
+
+# Scanning a monorepo in parts
+
+Monorepos, by definition, contain a large amount of code. The code is also often divided into to different components or modules, despite all existing in the same repository.
+
+As such, it can be helpful to scan a monorepo in parts for multiple reasons:
+
+1. Improving scan performance in CI and reducing CI run times
+2. Logically splitting up the monorepo into multiple components to help with managing findings
```

Comment:
```suggestion
* Improve scan performance in CI and reduce CI run times
* Logically split the monorepo to simplify managing findings
```

Reply:
Waiting on the other convo above to decide how to change this